src/components/SearchModal/index.jsx

```diff
@@ -74,7 +75,7 @@ const SearchModal = ({
 
 SearchModal.defaultProps = {
   appNodeId: 'App',
-  buttonSize: '',
+  buttonSize: null,
```

Comment:
This fixes a dev console error. The button component from CMS Design System doesn't like the empty string, only small or large. Passing null skips that prop-type issue.